src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts

```diff
@@ -55,6 +55,9 @@ import { formatMessageForTerminal } from 'vs/platform/terminal/common/terminalSt
 import { GroupKind } from 'vs/workbench/contrib/tasks/common/taskConfiguration';
 import { Codicon } from 'vs/base/common/codicons';
 
+const taskShellIntegrationStartSequence = '\x1b]633;A\x07' + '\x1b]633;P;Task=\x07' + '\x1b]633;B\x07';
+const taskShellIntegrationOutputSequence = '\x1b]633;C\x07';
```

Comment:
Not needed now but we may want to expose helpers for this in the terminal if we end up doing this sort of thing elsewhere.